### PR TITLE
단축 URL 구현

### PR DESCRIPTION
**Pull request by devunt**

### CODE REVIEW

**Comment by devunt** [comment]
Current dependencies on/for this PR:
* main
  * **PR #318** <a href="https://app.graphite.dev/github/pr/penxle/penxle/318" target="_blank">Graphite</a> 
    * **PR #319** <a href="https://app.graphite.dev/github/pr/penxle/penxle/319" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/319?utm_source=stack-comment).

File: apps/penxle.com/src/lib/server/graphql/schemas/post.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { init } from '@paralleldrive/cuid2';
```
Comment (robin-maki) [inline apps/penxle.com/src/lib/server/graphql/schemas/post.ts:1]: P3:
```suggestion
import { init as cuid } from '@paralleldrive/cuid2';
```
같은 식으로 cuid 함수명을 명확하게 하는 건 어떤가요?

File: apps/penxle.com/src/lib/server/graphql/schemas/post.ts [hunk 2]
```diff
@@ -404,6 +406,7 @@ builder.mutationFields((t) => ({
         create: {
           id: postId,
           permalink: customAlphabet('123456789', 12)(),
+          shortlink: init({ length: 6 })(),
```
Comment (robin-maki) [inline apps/penxle.com/src/lib/server/graphql/schemas/post.ts:409]:
```suggestion
          shortlink: cuid({ length: 6 })(),
```

**Comment by devunt** [comment]
### Merge activity

* **Nov 5, 9:47 PM**: @devunt started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/319).

**Pull request merged**